RecoTauTag/RecoTau/src/AntiElectronIDMVA6.cc

```diff
@@ -949,18 +975,32 @@ double AntiElectronIDMVA6::MVAValue(const pat::Tau& theTau, const pat::Electron&
   Int_t TauSignalPFGammaCandsIn = GammasPtInSigCone.size();
   Int_t TauSignalPFGammaCandsOut = GammasPtOutSigCone.size();
   Float_t TauVisMassIn = (pfGammaSum + pfChargedSum).mass();
-
-  Float_t TauPhi = theTau.phi();
- 
-  if ( theTau.phiAtEcalEntrance() != -99. ) {
-    TauPhi = theTau.phiAtEcalEntrance();
+  Float_t TauPhi = -99.;
+  if ( !usePhiAtEcalEntranceExtrapolation ) {
```

Comment:
is this conditional correct?
In all other cases in this file this branch picks up ```.phiAtEcalEntrance ``` and in the "else" is uses atECalEntrance method.
This part here is the inverse.